recipe-server/client/control_new/state/actions/selectors.js

```diff
@@ -1,5 +1,13 @@
-/* eslint import/prefer-default-export: "off" */
-
 export function getAction(state, id, defaultsTo = null) {
-  return state.app.actions.items.get(id, defaultsTo);
+  // Convert string IDs to integers, since the state is keyed by ints.
```

Comment:
It's called in `RecipeForm`'s connect, which passes `props.form.getFieldValue('action_id')` as the `id` param. I suspect that since we're pulling from Ant's `form` for the value, we could _possibly_ get a numeric value.

I think it makes sense to leave the type parsing in here, but I'm open to counterpoints.